projects/lib/src/oauth-service.ts

```diff
@@ -282,6 +282,15 @@ export class OAuthService extends AuthConfig implements OnDestroy {
         return lcUrl.startsWith('https://');
     }
 
+    protected assertUrlNotNullAndCorrectProtocol(url: string | undefined, description: string) {
+      if (url) {
+        throw new Error(`'${description}' should not be null`);
+      }
+      if (!this.validateUrlForHttps(url)) {
+        throw new Error(`'${description}' must use HTTP or config value for property requireHttps must allow HTTP.`);
```

Comment:
I think the original description was also somewhat confusing, but still more accurate. The old one allowed for two options:

- endpoint must use _https_; or...
- `requireHttps` must allow using http (without ssl/tls)

So either we should change it back to the old wording, or perhaps go along these lines:

```
throw new Error(`'${description}' must use https (with ssl/tls), or config value for property requireHttps must be set to false and allow http (without ssl/tls)`);

Reply:
adapted it.